src/components/EventItemAuthor.jsx

```diff
@@ -0,0 +1,14 @@
+import { requestUser } from "@/lib/fetchAPI";
+
+export const EventItemAuthor = async ({ userid }) => {
+  let data;
+  try {
+    data = await requestUser(userid);
+  } catch (error) {
+    console.error("An error occurred while fetching the user:", error);
```

Comment:
Include the id of the user when logging the error, so that we know which user causes this error.

- [ ] Example: `console.error("...", userid, error)`